CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_salesforce v1.1.1
+[PR #56](https://github.com/fivetran/dbt_salesforce/pull/56) includes the following updates:
+## Bugfix
+- Updated the logic for model `int_salesforce__date_spine` to reference the `stg_*` staging models instead of the source tables.
+  - This was necessary since the staging models account for multiple spellings of column names while the source tables do not.
```

Comment:
Can you also add either another bullet here or an under the hood entry that now that model has a depends on statement and also leverages the flags.WHICH. More for our purposes so we know how this issue was addressed and some underlying impacts.

Reply:
Updated!